multi-node-k8s.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# TODO: disable debugging
+set -x
+set -e -o pipefail
+
+# get latest stable version, strip the leading "v"
+K8S_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt | sed 's/^v//')"
+
+# enable additional sources for yum
+# (epel for ansible and golang)
+yum -y install epel-release
+
+# Install additional packages
+yum -y install \
+	qemu-kvm \
+	qemu-kvm-tools \
+	qemu-img \
+	libvirt \
+	ansible \
+	golang
+
+# install kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/"$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"/bin/linux/amd64/kubectl
```

Comment:
you can use the `${K8S_VERSION}` here, just make sure to prepend the "v" in the url

Reply:
Oh my bad, I missed it here. Thanks